scripts/qemu-wrapper

```diff
@@ -187,6 +187,12 @@ def main(argv):
     while n < len(qemu_args):
         p = qemu_args[n]
 
+        if p == "-netdev":
```

Comment:
Is the single dash correct? We mix `-flag` and `--flag` in the cli?

Reply:
The single dash is correct. QEMU upstream uses `-flag` only.

Any references to `--flag` are legacy qemu-trad bits which don't get used in the final invocation.